internal/controller/operator_configuration_controller.go

```diff
@@ -330,8 +330,9 @@ func (r *OperatorConfigurationReconciler) applyApiAccessSettings(
 
 		}
 	} else {
-		logger.Info("The API endpoint setting required for managing dashboards or check rules via the operator is " +
-			"missing or has been removed, the operator will not update dashboards nor check rules in Dash0.")
+		logger.Info("The API endpoint setting required for managing dashboards, check rules, synthetic checks and	 " +
+			"views via the operator is missing or has been removed, the operator will not update these resources " +
+			"in Dash0.")
```

Comment:
^ this is unrelated to the readiness check change. Managing synthetic checks and views with the operator has recently been implemented and these messages were not up to date yet.